website/public/js/services/analyticsServices.js

```diff
@@ -119,17 +119,19 @@ function analyticsFactory(User) {
   }
 
   function updateUser(properties) {
-    if (typeof properties === 'undefined') properties = {};
```

Comment:
Wasn't sure why you were testing specifically for undefined on these. Let me know if there was a reason.